editor/src/clj/editor/defold_project.clj

```diff
@@ -936,12 +1184,12 @@
   ([endpoint]
    (case (g/endpoint-label endpoint)
      (:build-targets) (project-resource-node? (g/now) (g/endpoint-node-id endpoint))
-     (:save-data :source-value) (project-file-resource-node? (g/now) (g/endpoint-node-id endpoint))
+     (:save-data :save-value) (project-file-resource-node? (g/now) (g/endpoint-node-id endpoint))
```

Comment:
`:source-value` is now trivial to produce, since we can get it from `user-data` in the project graph. This frees us up to cache `:save-value` instead. We're not really using more memory since it is also referenced from the `save-data`, but it is commonly used in connections whereas `save-data` is only really used for dirty-checks in the save system.